tests/fixtures/expectations/topology_child_tree.json

```diff
@@ -44,38 +44,6 @@
           },
           "tooltip": "Logistic UI Pod",
           "type": "KubernetesPod"
-        },
-        {
-          "agent_id": "00000000-0000-0000-0000-000000000000",
-          "icon": "icon-kubernetes-pod",
-          "id": "018681ff-80ed-d10d-21ef-c74f152b085b",
-          "is_leaf": false,
-          "name": "logistics-api-574dc95b5d-mp64w",
-          "parent_id": "018681ff-559f-7183-19d1-7d898b4e1413",
-          "parents": [
-            "018681fe-f5aa-37e9-83f7-47b5b0232d5e"
-          ],
-          "status": "healthy",
-          "summary": {
-            "healthy": 1
-          },
-          "type": "KubernetesPod"
-        },
-        {
-          "agent_id": "00000000-0000-0000-0000-000000000000",
-          "icon": "icon-kubernetes-pod",
-          "id": "018681ff-b6c1-a14d-2fd4-8c7dac94cddd",
-          "is_leaf": false,
-          "name": "logistics-ui-676b85b87c-tjjcp",
```

Comment:
@moshloop After the direct children change, there might be bugs/regressions which were copied over in some fixtures as well

For eg. here components are being repeated